.gitignore

```diff
@@ -1,2 +1,5 @@
 *build
 *.cache
+/engine/cmake-build-debug/
+/runtime/cmake-build-debug/
+/.idea/
```

Comment:
Please don't put editor-specific stuff in .gitignore. If you feel like you might accidentally commit these, put them in global .gitignore.